x/bank/client/rest/query.go

```diff
@@ -63,6 +64,8 @@ func QueryBalancesRequestHandlerFn(clientCtx client.Context) http.HandlerFunc {
 // HTTP request handler to query the total supply of coins
 func totalSupplyHandlerFn(clientCtx client.Context) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		client.AddDeprecationHeaders(w)
```

Comment:
Discovered that this route is not covered in Swagger docs, should it be added?

Reply:
No